src/store/modules/referent.js

```diff
@@ -0,0 +1,79 @@
+import axios from 'axios'
+export default ({
+  namespaced: true,
+  state: {
+    resources: [
+      { url: 'http://google.com', categories: ['searcher', 'docs'], id: 1 },
```

Comment:
If we have the integration we can remove those mock data here and get that data from the backend